src/wfs/src/main/java/org/geoserver/wfs/GetFeature.java

```diff
@@ -703,7 +703,11 @@ private BigInteger updateTotalCount(
             long total = getTotalCount(totalCountExecutors);
             totalCount = BigInteger.valueOf(total);
         } else {
-            if (SystemUtils.IS_JAVA_11 || SystemUtils.IS_JAVA_1_8) {
+            if (SystemUtils.IS_JAVA_21
+                    || SystemUtils.IS_JAVA_17
+                    || SystemUtils.IS_JAVA_11
+                    || SystemUtils.IS_JAVA_1_8) {
```

Comment:
Java 8 is no longer a supported platform

Reply:
That's a good point indeed. Perhaps this is more compact:

```
     if (SystemUtils.isJavaVersionAtMost(SystemUtils.IS_JAVA_21)
```